mmv1/third_party/terraform/tests/resource_composer_environment_test.go.erb

```diff
@@ -27,6 +27,16 @@ func init() {
 	})
 }
 
+func allComposerServiceAgents() []string {
```

Comment:
Nit: Could this be a const instead of a function?

Reply:
No, go does not support const slices and an array will be awkward if we ever add or remove elements.

Reply:
Oh wow, that's unfortunate! Seems like the function is the best option then, thanks for explaining.